mlflow/pyfunc/model.py

```diff
@@ -43,6 +43,7 @@
 CONFIG_KEY_PYTHON_MODEL = "python_model"
 CONFIG_KEY_CLOUDPICKLE_VERSION = "cloudpickle_version"
 _SAVED_PYTHON_MODEL_SUBPATH = "python_model.pkl"
+FLAVOR_NAME = "python_function"
```

Comment:
We have the same `FLAVOR_NAME` here:

https://github.com/mlflow/mlflow/blob/071adcf371b057d5172e43cfd20e020974a6682d/mlflow/pyfunc/__init__.py#L514